ext/nnstreamer/tensor_filter/tensor_filter_caffe2_core.cc

```diff
@@ -262,11 +263,17 @@ Caffe2Core::run (const GstTensorMemory * input, GstTensorMemory * output)
 #if (DBG)
   gint64 start_time = g_get_real_time ();
 #endif
-  for (i = 0; i < inputTensorMeta.num_tensors; i++){
+
+  for (i = 0; i < inputTensorMeta.num_tensors; i++) {
     Tensor *inputTensor = inputTensorMap.
                             find(inputTensorMeta.info[i].name)->second;
-    inputTensor->ShareExternalPointer ((float*) input[i].data);
```

Comment:
Is this the only effective change of this PR?

Reply:
line 276 is major change (in switch cond, output > input)